aea/configurations/base.py

```diff
@@ -2045,20 +2045,6 @@ def __init__(self, **args):
         """Initialize a speech_act content configuration."""
         super().__init__()
         self.args = args  # type: Dict[str, str]
-        self._check_consistency()
-
-    def _check_consistency(self):
-        """Check consistency of the args."""
-        for content_name, content_type in self.args.items():
-            if not isinstance(content_name, str) or not isinstance(content_type, str):
```

Comment:
The first part of the if was inconsequential/unreachable (if not string, yaml converter would raise error.)
The seance part added to the validator in the generator.